test/e2e/util_test.go

```diff
@@ -73,25 +74,25 @@ func newNamespaceCleaner(namespace string) *namespaceCleaner {
 }
 
 // notifyOnFailure checks if a test has failed or cleanup is true before cleaning a namespace
-func (c *namespaceCleaner) NotifyTestComplete(t GinkgoTInterface, cleanup bool) {
-	if t.Failed() {
+func (c *namespaceCleaner) NotifyTestComplete(t GinkgoTestDescription, cleanup bool) {
```

Comment:
Does this method need to accept `GinkgoTestDescription` as a parameter? `CurrentGinkgoTestDescription` can be called from within this method to get the same value.